apps/speed.c

```diff
@@ -933,11 +1459,28 @@ int speed_main(int argc, char **argv)
         goto end;
     }
 
+    if (async_jobs > 0) {
+        if (!ASYNC_init(1, 0, 0)) {
```

Comment:
Since the operation is repeated *many* times the cost of initialization should be negligible.
However at this point we know the number of jobs so I could potentially initialize the pool.